frontend/app/src/components/import/GroupedImport.vue

```diff
@@ -39,56 +47,72 @@ import { get } from '@vueuse/core';
 import { computed, ref } from 'vue';
 import { useI18n } from 'vue-i18n-composable';
 import AdaptiveWrapper from '@/components/display/AdaptiveWrapper.vue';
+import BinanceImport from '@/components/import/BinanceImport.vue';
+import BisqImport from '@/components/import/Bisq.vue';
+import BlockFiImport from '@/components/import/BlockFiImport.vue';
+import CointrackingImport from '@/components/import/CointrackingImport.vue';
+import CryptoComImport from '@/components/import/CryptoComImport.vue';
+import CustomImport from '@/components/import/CustomImport.vue';
+import NexoImport from '@/components/import/NexoImport.vue';
+import ShapeshiftImport from '@/components/import/ShapeshiftImport.vue';
+import UpholdImport from '@/components/import/UpholdImport.vue';
+import i18n from '@/i18n';
```

Comment:
In this components `vue-i18n-composable` is already used, please don't import `@/i18n` directly but use the `tc` or `t` from from `useI18n`

Reply:
Sorry, I think I missed it when rebasing